recipes/jpeg-compressor/all/conanfile.py

```diff
@@ -10,7 +10,7 @@ class JpegCompressorConan(ConanFile):
     homepage = "https://github.com/richgel999/jpeg-compressor"
     topics = ("conan", "jpeg", "image", "compression", "decompression")
     url = "https://github.com/conan-io/conan-center-index"
-    license = "Public Domain", "Apache-2.0"
+    license = "Unlicense", "Apache-2.0"
```

Comment:
https://github.com/richgel999/jpeg-compressor/blob/aeb7d3b463aa8228b87a28013c15ee50a7e6fcf3/jpgd_idct.h#L1

```suggestion
    license = "Unlicense", "Apache-2.0", "MIT"
```